packages/pigeon/pigeons/core_tests.dart

```diff
@@ -83,9 +83,10 @@ class AllNullableTypes {
 }
 
 // A class for testing nested object handling.
-class AllNullableTypesWrapper {
-  AllNullableTypesWrapper(this.values);
-  AllNullableTypes values;
+class AllClassesWrapper {
+  AllClassesWrapper(this.allNullableTypes, this.allTypes);
+  AllNullableTypes allNullableTypes;
+  AllTypes? allTypes;
```

Comment:
Could you explicitly comment that we have both here in order to test handling of both nullable and non-nullable class fields? It's non-obvious (and potentially confusing since the "nullable" naming refers to the fields of the class itself, and doesn't match the type here).

Reply:
I can rename these variables as well, if you think that will help.

Reply:
I think just a comment is fine; while `nonNullableAllNullableTypes` and `nullableAllTypes` are a bit more descriptive, I don't think it's probably worth the effort of renaming everywhere. The main thing is that we should make sure it's very clear via comment that this is intentional, and why we need both kinds of fields.